backend/tracim_backend/lib/core/event.py

```diff
@@ -314,6 +308,9 @@ def create_event(
             entity_subtype=entity_subtype,
             fields=fields,
             workspace_id=fields.get("workspace", {}).get("workspace_id"),
+            content_id=fields.get("content", {}).get("content_id"),
+            parent_id=fields.get("content", {}).get("parent_id"),
+            author_id=fields.get("author", {}).get("user_id") if fields.get("author", {}) else None,
```

Comment:
```suggestion
            author_id=fields.get("author", {}).get("user_id") if fields.get("author", {}) else None,
```
I need to do some investigation about why i do need the if/else.

Reply:
Verified in TLM specification.